pp/integrations/plugin-packs/procedures/applications-sailpoint-identitynow-restapi.md

```diff
@@ -0,0 +1,389 @@
+---
+id: applications-sailpoint-identitynow-restapi
+title: SailPoint IdentityNow Rest API
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Pack assets
+
+### Templates
+
+The Monitoring Connector **SailPoint IdentityNow Rest API** brings a host template:
+
+* **App-Sailpoint-Identitynow-Restapi-custom**
+
+The connector brings the following service templates (sorted by the host template they are attached to):
+
+<Tabs groupId="sync">
+<TabItem value="App-Sailpoint-Identitynow-Restapi-custom" label="App-Sailpoint-Identitynow-Restapi-custom">
+
+| Service Alias | Service Template  | Service Description |
+|:--------------|:------------------|:--------------------|
```

Comment:
bug?

Reply:
there is no service attached by default (normal)